client/src/client_sync/v17/mining.rs

```diff
@@ -16,7 +16,7 @@ macro_rules! impl_client_v17__getblocktemplate {
         impl Client {
             pub fn get_block_template(
                 &self,
-                request: &$crate::client_sync::TemplateRequest,
+                request: &TemplateRequest,
```

Comment:
This confused me for a while. I'm going to ack and merge this and iterate on top of it. Thanks man.